app/src/main/kotlin/edu/stanford/bdh/engagehf/medication/ui/MedicationViewModel.kt

```diff
@@ -55,14 +64,36 @@ class MedicationViewModel @Inject internal constructor(
                     )
                 }
             }
+
             is Action.InfoClicked -> {
-                // TODO
+                viewModelScope.launch {
+                    // it seems that this is not entirely consistent, so if action.videoPath starts without "/", it will be added
+                    val correctedPath =
```

Comment:
I would remove this logic here and adapt both regexes in the mapper to handle the initial slash as optional, feel free to pass `action.videoPath` here directly

Reply:
great idea!